_cards_adapter/nmt-reform-rpicm4-som.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: card
```

Comment:
This would instead be a board (even though technically it just adapts to a new type of SoM). Can you update the layout for it and name of the picture?

Reply:
Yep! I'll do that. Should I move it over to the boards folder as well, I'm assuming?